meson_options.txt

```diff
@@ -4,7 +4,8 @@ option('libseat', type: 'feature', value: 'auto', description: 'Enable support f
 option('xcb-errors', type: 'feature', value: 'auto', description: 'Use xcb-errors util library')
 option('xcb-icccm', type: 'feature', value: 'auto', description: 'Use xcb-icccm util library')
 option('xwayland', type: 'feature', value: 'auto', yield: true, description: 'Enable support for X11 applications')
-option('x11-backend', type: 'feature', value: 'auto', description: 'Enable X11 backend')
 option('examples', type: 'boolean', value: true, description: 'Build example applications')
 option('icon_directory', description: 'Location used to look for cursors (default: ${datadir}/icons)', type: 'string', value: '')
 option('xdg-foreign', type: 'feature', value: 'auto', description: 'Enable xdg-foreign protocol')
+option('backends', type: 'combo', choices: ['auto', 'drm', 'noop', 'headless', 'libinput', 'x11', 'wayland'],
+       description: 'Select the backend support')
```

Comment:
This is missing an option to toggle the GLES2 renderer.

Reply:
Well, it is not harm to keep such stub code, anyway it would link to a GLES2 library anyway.
If that platform doesn't have a GPU, it won't be able to find a library, the link check would disable the renderer there.

Reply:
Let me rephrase this: we can't have `dependency('glesv2', optional: true)` without a way for the user to explicitly enable or disable it. Some users may have the library installed but may still want to disable the GLES2 renderer. Auto-detection is nice for end users, but cause issues in CI and automated build environments.